index.html

```diff
@@ -14,7 +14,7 @@
     <link rel="icon" type="image/png" href="/favicon.png" />
     <link rel="stylesheet" href="/css/main.css">
 
-    <script src="bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+    <script src="https://cdn.rawgit.com/webcomponents/webcomponentsjs/v0.7.24/webcomponents-lite.js"></script>
```

Comment:
Don't think you should change this. I noticed webcomponentsjs isn't shipping a minified version anymore, but you could just remove `.min`.

You can change webcomponentsjs versions via the bower file.